src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -147,10 +147,21 @@ def new_normalize_text(s):
         self._register_help_loaders()
         self._name_to_content = {}
 
-    # override
     def show_help(self, cli_name, nouns, parser, is_group):
         self.update_loaders_with_help_file_contents(nouns)
-        super(AzCliHelp, self).show_help(cli_name, nouns, parser, is_group)
+
+        import colorama
+        colorama.init(autoreset=True)
+        delimiters = ' '.join(nouns)
+        help_file = self.command_help_cls(self, delimiters, parser) if not is_group \
+            else self.group_help_cls(self, delimiters, parser)
+        help_file.load(parser)
+        if not nouns:
+            help_file.command = ''
+        else:
+            help_file.examples = AzCliHelp.check_for_new_examples(help_file.command, help_file.examples)
```

Comment:
Let's simplify the function signature.
```suggestion
            AzCliHelp.check_for_new_examples(help_file)
```

Reply:
Updated!